messaging/src/test/java/org/axonframework/eventhandling/GapAwareTrackingTokenTest.java

```diff
@@ -205,12 +205,43 @@ void tokenCoversOther() {
         assertFalse(token3.covers(token7));
     }
 
+    @Test
+    void tokenSameAsOther() {
```

Comment:
I don't think the test name aligns with the test method. Which I assume you intended to, as the original method name was `same`.